### PR TITLE
Bug report status

### PR DESCRIPTION
**Pull request by Drwildy**
Bug report backend unit tests, and new endpoints for finding pending bugs, approving bugs, and denying bugs.

### CODE REVIEW

File: BugBounty/src/test/java/com/revature/service/ApproveDenyBugServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+	}
+	
+	@Test
+	public void testUpdateBugReportStatus() {
+		//Mockito.doNothing().when(bugReportRepository).updateStatus(8, "open");
+		//Mockito does nothing the first time for free
+		bugReportService.updateBugReportStatus(bugReport);
+	}
+	
+	@Test
```
Comment (russchristina) [inline BugBounty/src/test/java/com/revature/service/ApproveDenyBugServiceTest.java:44]: Is there some business logic present in the deleteBugReport method in your service layer that you're intending to test here?

Comment (Drwildy) [reply BugBounty/src/test/java/com/revature/service/ApproveDenyBugServiceTest.java:44]: I was wanting to test that the service layer calls the repository deleteById. Wasn't sure how to mock that. Is there a btter way, same with the UpdateBugReportStatus? -Devin

Comment (russchristina) [reply BugBounty/src/test/java/com/revature/service/ApproveDenyBugServiceTest.java:44]: You may want to use Mockito's "verify" for this task.

Comment (Drwildy) [reply BugBounty/src/test/java/com/revature/service/ApproveDenyBugServiceTest.java:44]: Ok I will resubmit

**Comment by Drwildy** [comment]
Closed to fix testing void method

**Pull request closed without merging**